packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx

```diff
@@ -0,0 +1,735 @@
+import { Link } from '../atoms';
+import { FormCard, LabeledMultiSelect } from '../molecules';
+import { noop } from '../utils';
+
+type TeamCreateOutputExtraInformationProps = {
+  values: string[];
+  onChange?: (values: string[]) => void;
+};
+
+// The list is auto generated // TODO: Where should we put this?
+const tags = [
```

Comment:
we already have these in the code at `expertiseAndResourceSuggestions`

Reply:
Ah thank you - I didn't think of searching for them.

The list seems to be updated:
- ~300 new items
- There are many renames
- Some deletions

I'd propose to reuse them and update them in a separate issue.

Reply:
Follow up: The list should be separate. I will move it next to `expertiseAndResourceSuggestions` with a note that the duplication is intentional